website/docs/docs/build/cumulative-metrics.md

```diff
@@ -31,10 +32,11 @@ metrics:
     type: cumulative # Required
     label: The value that will be displayed in downstream tools # Required
     type_params: # Required
-      fill_nulls_with: Set the value in your metric definition instead of null (such as zero) # Optional
-      measure: The measure you are referencing # Required
-      window: The accumulation window, such as 1 month, 7 days, 1 year. # Optional. Cannot be used with grain_to_date
-      grain_to_date: Sets the accumulation grain, such as month will accumulate data for one month, then restart at the beginning of the next.  # Optional. Cannot be used with window
+      measure: 
```

Comment:
is this right? it's measure not 'measures' with an s right?

Reply:
Yup!